styles/settings-view.less

```diff
@@ -7,6 +7,7 @@
 
 .settings-view {
   display: flex;
+  overflow: auto;
```

Comment:
How about instead of making the whole settings-view scrollable, limit it to only `.settings-view .panels`. Then the left navigation stays fixed. Like this:

```diff
.settings-view .panels {
+  overflow: auto;
}

.settings-view .panels .panels-item {
-  overflow: auto;
}
```